include/sir/platform.h

```diff
@@ -82,6 +89,10 @@
 #  define SANITIZE_SUPPRESS(str)
 # endif
 
+# if HAS_FEATURE(safe_stack) && !defined(SIR_NO_PLUGINS)
+#  error "linking dynamically shared objects with SafeStack is unsupported; disable SafeStack or enable SIR_NO_PLUGINS"
```

Comment:
Now that I think of it, "linking dynamically shared objects" is redudnant (dynamic and shared imply each other), and if you dynamically load them it's technically not "linking." I would put "loading libraries dynamically while also using SafeStack is..."

Reply:
I just used the terminology Clang uses in the documentation:

> SafeStack supports linking statically modules that are compiled with and without SafeStack. An executable compiled with SafeStack can load dynamic libraries that are not compiled with SafeStack. At the moment, compiling dynamic libraries with SafeStack is not supported.

> Linking a DSO with SafeStack is not currently supported.

Reply:
I can still change it if you want.